src/ru/skypro/Main.java

```diff
@@ -1,8 +1,30 @@
 package ru.skypro;
 
+import java.util.Locale;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+	// Задание 1
+        String firstName = "Ivan";
+        String middleName = "ivanovich";
+        String lastName = "Ivanov";
+        String space = " ";
+        String fullName = lastName + space + firstName + space + middleName;
```

Comment:
можете также посмотреть в сторону форматируемых строк
```suggestion
        String fullName = String.format("%s %s %s", lastName, firstName, middleName);
```

Reply:
Спасибо, Олег за подсказку. Попробую переделать